config.yaml

```diff
@@ -106,20 +106,24 @@ miniAMR:
     clean: [ 'git clean -fx' ]
 
 XSBench:
-    fetch: 'git clone https://github.com/ANL-CESAR/XSBench.git'
+    fetch: 'git clone https://github.com/ANL-CESAR/XSBench.git;
```

Comment:
Please add the command to clone/checkout the specific commit that this patch works with. Also for RSBench fetch. We should do it for every repo that needs a patch.